lib/ansible/modules/network/netvisor/pn_fabric_local.py

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/python
+# Copyright: (c) 2018, Pluribus Networks
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = """
+---
+module: pn_fabric_local
+author: "Pluribus Networks (@rajaspachipulusu17)"
+version_added: "2.9"
+short_description: CLI command to modify fabric-local
+description:
+  - This module can be used to modify fabric local information.
+options:
+  pn_cliswitch:
+    description:
+      - Target switch to run the CLI on.
+    required: False
+    type: str
+  state:
+    description:
+      - State the action to perform. Use C(update) to modify the fabric-local.
+    required: True
+    type: str
+    choices: ['update']
```

Comment:
Also, this module has only one choice `update`, if yes then can this module be part of any other parent module _(as only update call is being handled here)_

Reply:
This cant be merged. It will be used when ever required. The previous command will be done

Reply:
ok